components/dashboard/src/projects/NewProject.tsx

```diff
@@ -300,15 +276,24 @@ export default function NewProject() {
         const showSearchInput = !!repoSearchFilter || filteredRepos.length > 0;
 
         const renderRepos = () => (<>
-            {!isBitbucket() && <p className="text-gray-500 text-center text-base">Select a Git repository on <strong>{selectedProviderHost}</strong>. (<a className="gp-link cursor-pointer" onClick={() => setShowGitProviders(true)}>change</a>)</p>}
-            <div className={`mt-10 border rounded-xl border-gray-100 dark:border-gray-800 flex-col`}>
-                <div className="px-8 pt-8 flex flex-col space-y-2" data-analytics='{"label":"Identity"}'>
+            <p className="text-gray-500 text-center text-base">Projects allow you to manage prebuilds and workspaces for your repository. <a href="https://www.gitpod.io/docs/teams-and-projects" rel="noopener" className="gp-link">Learn more</a></p>
+            <p className="text-gray-500 text-center text-base mt-12">Select account on <b>{selectedProviderHost}</b> (<a className="gp-link cursor-pointer" onClick={() => setShowGitProviders(true)}>change</a>)</p>
+            <div className={`mt-2 flex-col w-96`}>
+                <div className="px-8 pt-5 flex flex-col space-y-2" data-analytics='{"label":"Identity"}'>
                     <ContextMenu classes="w-full left-0 cursor-pointer" menuEntries={getDropDownEntries(accounts)}>
                         <div className="w-full">
-                            {icon && (
-                                <img src={icon} className="rounded-full w-6 h-6 absolute top-1/4 left-4" />
+                            {!selectedAccount && user && user.name && user.avatarUrl && (
+                                <>
+                                    <img src={user?.avatarUrl} className="rounded-full w-6 h-6 absolute top-1/4 left-3" />
+                                    <input className="w-full px-12 cursor-pointer font-semibold" readOnly type="text" value={user?.name}></input>
+                                </>
+                            )}
+                            {selectedAccount && (
+                                <>
+                                    <img src={icon ? icon : ""} className="rounded-full w-6 h-6 absolute top-1/4 left-3" />
```

Comment:
Totally better to fix the underlying issue, but thought we would want to have an intermediate thing. But I agree with you to leave it 👏🏼

Reply:
Let's cut some corners to ship this! 🧀 

Looping in @jldec in case this (https://github.com/gitpod-io/gitpod/issues/5330) is something we'd like to prioritize soon.